src/resources/lang/de/bookmark.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/*
+ * This file is part of SeAT
+ *
+ * Copyright (C) 2015, 2016, 2017, 2018, 2019  Leon Jacobs
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+return [
+    'created'     => 'Erstellt',
+    'label'       => 'Kennzeichnung',
```

Comment:
@ppfeufer translate that one with `Bezeichnung`
which one is better ? :)

Reply:
Bezeichnung, since it is the name of the bookmark. Kennzeichnung would be a mark on a technical blueprint or something like that.

Both german words can be translated to label though, but one is more the descriptive type (Bezeichnung) and one is more the technical type (Kennzeichnung).